Contributing.md

```diff
@@ -0,0 +1,59 @@
+
+## How to Contribute
+
+To contribute to DeliverEase, please follow these guidelines:
+
+1. Fork the repository to your GitHub account.
+
+2. Create a new branch from the `main` branch. Use a descriptive name for your branch that reflects the nature of your contribution.
+
+3. Make your changes to the codebase or documentation in your branch. Be sure to follow the coding style and conventions used in the project.
+
+4. Write clear and concise commit messages for your changes.
+
+5. Test your changes thoroughly to ensure they work as expected.
+
+6. Update or create new tests where applicable.
+
+7. Ensure that all existing tests pass successfully before submitting your changes.
+
+8. Push your branch to your forked repository.
+
+9. Create a pull request (PR) from your branch to the `main` branch of the main repository.
+
+10. Provide a detailed description of your changes in the pull request, including the problem you are solving, the approach you took, and any relevant information for reviewers.
+
+11. Be responsive to feedback and comments on your pull request. If requested changes are needed, make the necessary adjustments.
+
+12. Once your pull request is approved and meets the project's quality standards, it will be merged into the main repository.
+
+## Code Style and Guidelines
+
+To maintain a consistent codebase, we follow specific code style and guidelines. Please adhere to the following conventions when contributing:
+
+- Python code should be properly written.
+- HTML and CSS should follow best practices and conventions for web development.
+- Write clear and well-documented code.
+- Use meaningful variable and function names.
+- Include comments where necessary to explain complex logic or processes.
+
+## Comment on Issues
+
+You can actively participate by commenting on the open issues. If you think you can solve a particular issue, comment "Please assign this issue to me," and one of the maintainers will assign the issue to you. Once assigned, you can start working on the problem and create a pull request once your solution is ready.
+
+## Reporting Issues
+
+If you come across any issues, bugs, or have suggestions for improvements, please report them using GitHub Issues. When reporting issues, provide as much relevant information as possible, including steps to reproduce the problem, your operating system, browser details, etc.
+
+## Code of Conduct
+
+We expect all contributors to adhere to our Code of Conduct, which promotes a welcoming and inclusive community. Please be respectful and considerate of others' opinions and contributions.
+
+## License
+
+By contributing to DeliverEase, you agree that your contributions will be licensed under the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
This might need to be updated in case we decide to go with any other License.
cc: @Devansh-Singh-Parmar @N3v1

Reply:
I would separate the text from the license text. Just leave three
Free lines between the two texts and in the second line write --- , or indicate the "Thank you ..." text as a heading with ##.

Reply:
I am removing License section for now, please choose the license.

cc: @SID262000 @N3v1